routes/auth.ts

```diff
@@ -9,6 +9,10 @@ router.get("/github/login", auth.githubAuthLogin);
 
 router.get("/github/callback", auth.githubAuthCallback);
 
+router.get("/google/login", auth.googleAuthLogin);
+
+router.get("/google/callback", auth.googleAuthCallback);
+
```

Comment:
There is a middleware present to check for dev flag at route level, see if that can be used

Reply:
done